src/Aserto.AspNetCore.Middleware/Aserto.AspNetCore.Middleware.csproj

```diff
@@ -27,21 +27,21 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Aserto.Authorizer.V2.Client.Grpc" Version="0.0.5" />
-    <PackageReference Include="Aserto.Directory.V3.Client.Grpc" Version="0.31.2" />
-    <PackageReference Include="Google.Protobuf" Version="3.25.0-rc2" />
-    <PackageReference Include="Grpc.Core.Api" Version="2.49.0" />
-    <PackageReference Include="Grpc.Net.Client" Version="2.49.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="3.1.30" />
+    <PackageReference Include="Aserto.Authorizer.V2.Client.Grpc" Version="0.0.6" />
+    <PackageReference Include="Aserto.Directory.V3.Client.Grpc" Version="0.31.3" />
+    <PackageReference Include="Google.Protobuf" Version="3.27.2" />
+    <PackageReference Include="Grpc.Core.Api" Version="2.64.0" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.64.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="8.0.7" />
     <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.2" />
-    <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0" />
-    <PackageReference Include="MinVer" Version="4.2.0">
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Options" Version="8.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="8.0.0" />
+    <PackageReference Include="MinVer" Version="5.0.0">
```

Comment:
Please use  <PackageReference Include="MinVer" Version="3.1.0"> to align with the language binding projects